engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -100,6 +100,7 @@
   private static final long DELEGATION_TOKEN_RENEW_INTERVAL = Long.MAX_VALUE / 2;
   private static final long DELEGATION_TOKEN_REMOVER_SCAN_INTERVAL = 24 * 60 * 60 * 1000;
   private static final int NUMBER_MISSED_HEARTBEATS = 30;
+  private static final int MAX_CONTAINER_FAILURES_PER_NODE = 10;
```

Comment:
Agree, can the default be changed to 3 and provide ability for end user to overwrite the default value.